Orange/tests/test_ada_boost.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import numpy as np
+from Orange.data import Table
+from Orange.classification import TreeLearner
+from Orange.regression import TreeRegressionLearner
+from Orange.ensembles import SklAdaBoostLearner, SklAdaBoostRegressionLearner
+from Orange.evaluation import CrossValidation, CA, RMSE
+
+
+class SklAdaBoostTest(unittest.TestCase):
+    def test_adaboost(self):
+        table = Table("iris")
```

Comment:
Could you load those tables in a `setUp()` method once and then use that? I know it's not uniform practice across our tests, but with new ones, we really can _make better_.